### DISCUSSION TITLE
Welcome to ise Discussions!

### DISCUSSION THREAD

**Discussion by falkowich** [falkowich/ise#146] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pboertje** [comment]
Hi @falkowich,

Great to see ISE the code, very helpful when working with ISE.
For a case I'm looking in to GuestUsers. Is there any development on the Guestusers part of ISE?

**Reply by falkowich** [reply]
Hi @patrickboertje 

Not at the moment, but we are looking into the same thing here.
We need to atomagicate that part some more, so it's on the radar. But nothing has started yet.
If you have any ideas or code I would be happy to include it with a PR.

Or just ideas and we could see if it is possible to do the code and include it.

--
Regards Falk
